contrib/pai_vscode/README_zh_CN.md

```diff
@@ -35,7 +36,7 @@ OpenPAI VS Code Client 是一个 Visual Studio Code 的扩展组件，可以连
       ![添加集群](https://raw.githubusercontent.com/Microsoft/pai/master/contrib/pai_vscode/assets/add_cluster.png)
       
 
-4. 配置文件将会被打开，至少需要填入 username 和 password 字段。 完成后，点击右下角的 *Finish* 按钮。 注意，如果直接保存并关闭文件，则无法生效。
+4. 配置文件将会被打开，至少需要填入 username 和 password 字段。 完成后，点击右下角的 *完成* 按钮。 注意，如果直接保存并关闭文件，则无法生效。
```

Comment:
Do we really has the "完成" button instead of "Finish"? I mean have we made the I18N for the extension? if not, you might still want to use "Finish" here.

Reply:
In Chinese mode, there is a '完成' button, not 'Finish'.
![finish](https://user-images.githubusercontent.com/17357108/75664895-361a9c80-5cae-11ea-9b43-965299620e75.png)